### PR TITLE
{bio} [goolf-1.4.10] HTSeq-0.6.1p1 (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**
A user in my cluster requested this tool patched as explained here
http://seqanswers.com/forums/showthread.php?t=41531

he was hitting this error:

```
Error occured when processing SAM input (record #43311736 in file RIN10A_polyA.sorted.bam):
Maximum alignment buffer size exceeded while pairing SAM alignments.
[Exception type: ValueError, raised in __init__.py:671]
```

### CODE REVIEW

File: easybuild/easyconfigs/h/HTSeq/HTSeq-0.6.1p1-goolf-1.4.10-Python-2.7.5.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+homepage = 'http://www-huber.embl.de/users/anders/HTSeq/doc/overview.html'
+description = """HTSeq is a Python package that provides infrastructure to process data from high-throughput sequencing
+assays."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+patches = ['HTSeq-0.6.1p1.patch']
```
Comment (boegel) [inline easybuild/easyconfigs/h/HTSeq/HTSeq-0.6.1p1-goolf-1.4.10-Python-2.7.5.eb]: include a descriptive label, and use `%(version)s`:

```python
patches = ['HTSeq-%(version)s_max-buffer-size-30M.patch']
```

File: easybuild/easyconfigs/h/HTSeq/HTSeq-0.6.1p1.patch [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Increase buffer size in function pair_SAM_alignments_with_buffer to avoid this error:
+
+# Error occured when processing SAM input (record #43311736 in file RIN10A_polyA.sorted.bam):
+#  Maximum alignment buffer size exceeded while pairing SAM alignments.
+#  [Exception type: ValueError, raised in __init__.py:671]
+
+# reference: http://seqanswers.com/forums/showthread.php?t=41531
+
```
Comment (boegel) [inline easybuild/easyconfigs/h/HTSeq/HTSeq-0.6.1p1.patch]: include author?

File: easybuild/easyconfigs/h/HTSeq/HTSeq-0.6.1p1-goolf-1.4.10-Python-2.7.5.eb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'PythonPackage'
+
+name = 'HTSeq'
+version = '0.6.1p1'
```
Comment (boegel) [inline easybuild/easyconfigs/h/HTSeq/HTSeq-0.6.1p1-goolf-1.4.10-Python-2.7.5.eb]: use a (descriptive) `versionsuffix`?

File: easybuild/easyconfigs/h/HTSeq/HTSeq-0.6.1p1-goolf-1.4.10-Python-2.7.5.eb [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+versionsuffix = '_patched_max-buffer-size'
```
Comment (boegel) [inline easybuild/easyconfigs/h/HTSeq/HTSeq-0.6.1p1-goolf-1.4.10-Python-2.7.5.eb]: I would've used the same as in the patch...

```python
versionsuffix = '-max-buffer-size-30M'
...
patches = ['HTSeq-%(version)s%(versionsuffix)s.patch']

**Comment by pescobar** [comment]
test this please

File: easybuild/easyconfigs/h/HTSeq/HTSeq-0.6.1p1-goolf-1.4.10-Python-2.7.5.eb [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+patches = ['HTSeq-%(version)s%(versionsuffix)s.patch']
+
+python = 'Python'
+pythonver = '2.7.5'
+versionsuffix = '-%s-%s' % (python, pythonver)
```
Comment (boegel) [inline easybuild/easyconfigs/h/HTSeq/HTSeq-0.6.1p1-goolf-1.4.10-Python-2.7.5.eb]: @pescobar: `versionsuffix` is being overwritten here...

**Comment by boegel** [comment]
@pescobar: ping on making Jenkins happy? :)

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months, obsoleted by more recent PRs.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**